### PR TITLE
Make Act Title and Act Section optional

### PR DESCRIPTION
**Pull request by willbarton**
According to @cmc333333, the `act_title` and `act_section` arguments to the parser are only used in the external citations layer, which is never retrieved, so they're generally unused. This PR makes these two arguments optional. This means running, E, for example, goes from:

```shell
./build_from.py fr-notices/articles/xml/201/131/725.xml 12 15 1693
```

To:

```shell
./build_from.py fr-notices/articles/xml/201/131/725.xml 12
```

The result is that the JSON outputted to `layers/external_citations/1005/` looks like this:

```json
    "1005-1-b": [
        {
            "citation": [
                15, 
                1693
                0, 
                0
            ], 
            ....
        }
```

The README is updated to reflect this change (and the removal of notice_doc in #274).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,27 +148,41 @@ regulation E).
...
+section. Wherever the phrase "the Act" is used in the regulation, the 
+external link parser will treat it as "15 U.S.C. 1693".  The final rule 
+number is used to pull in section-by-section analyses and deduce which 
+notices were used to create this version of the regulation. It also 
+helps determine which notices to use when building additional versions 
+of the regulation. To find the document number, use the 
+[Federal Register](https://www.federalregister.gov/), finding the last, 
+effective final rule for your version of the regulation and copying the 
+document number from the meta data (currently in a table on the right 
+side).
```
Comment (cmc333333) [inline README.md]: Seems like the whole bit about final rule number (including its presence in the example command) can be removed?

**Comment by cmc333333** [comment]
Very nice. We'll get it down to zero arguments in no time!

**Comment by pkfec** [comment]
do we know when this change will be merged to the master branch?

**Comment by cmc333333** [comment]
ping

**Pull request closed without merging**